package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "teamsnap-ui",
+  "version": "0.1.0",
+  "description": "a CSS component library for TeamSnap",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/teamsnap/teamsnap-ui.git"
+  },
+  "author": "",
```

Comment:
Wasn't sure if there was a `design@teamsnap.com` alias or something that would be useful here.